classes/migration/install/LogMigration.php

```diff
@@ -49,10 +49,11 @@ public function up(): void
             $table->foreign('log_id', 'event_log_settings_log_id')->references('log_id')->on('event_log')->onDelete('cascade');
             $table->index(['log_id'], 'event_log_settings_log_id');
 
+            $table->string('locale', 14)->default('');
             $table->string('setting_name', 255);
             $table->mediumText('setting_value')->nullable();
-            $table->string('setting_type', 6)->comment('(bool|int|float|string|object)');
-            $table->unique(['log_id', 'setting_name'], 'event_log_settings_unique');
+            $table->string('setting_type', 6)->comment('(bool|int|float|string|object)')->nullable();
```

Comment:
I think I'd rather not introduce a `nullable` here, for consistency with the `setting_type` columns on other `_settings` tables. Feel free to force a `string` to restored entries, since that should work for all existing entries.

Reply:
Oоps, setting_type is not longer needed and should be removed - I made it nullable in my attempt to record custom log entries.